res/values/strings.xml

```diff
@@ -36,18 +36,10 @@
     <string name="menuItem_enableFramelimiter">Sync audio</string>
     <string name="menuItem_disableFramelimiter">Desync audio</string>
     <string name="menuItem_paks">Controller Paks</string> <!-- capitalize since trademark -->
-    <string name="menuItem_pak1_empty">Player 1: Empty</string>
```

Comment:
I know we shouldn't be screwing around with the strings.xml file too much, but I decided to make this one exception since it would be trivial to update the translations ourselves (thanks to the text before and after the colon being independent clauses).